documentation/src/docs/pravega-concepts.md

```diff
@@ -11,362 +11,266 @@ You may obtain a copy of the License at
+
+The data from the IoT sensor is extracted or read by a variety of applications (_e.g.,_ Java applications) and writes the read or fetched data to the tail (front) of the stream. Applications, such as a [Flink](https://flink.apache.org), can read from any point in the stream. Many applications can read and write the same stream in parallel. Elasticity, scalability, support for large volume of
+stream data and applications are the highlights of Pravega's design. More information on read and write operations in the streams will be discussed in the [Readers and Writers](#writer) section.
 
 ## Events
 
-Pravega's client API allows applications to read and write data in Pravega in terms of an Event.  An Event is a
-set of bytes within a Stream.  An Event could be as simple as a small
-number of bytes containing a temperature reading from an IoT sensor composed of
-a timestamp, a metric identifier and a value.  An Event could be web log data
-associated with a user click on a website.  Events can be anything you can
-represent as a set of bytes.  Applications make sense of Events using
-standard Java serializers and deserializers, allowing them to read and write
-objects in Pravega using similar techniques to reading and writing objects from
+Pravega's client API allows applications to read and write data in Pravega using **Events**.  An event is represented as a set of bytes within a stream. For example, an event could be as simple as a small number of bytes containing a temperature reading from an IoT sensor composed of
+a timestamp, a metric identifier and a value or an event could be web log data
+associated with a user click on a website. Applications make sense of events using
+standard Java **Serializers** and **Deserializers**, allowing them to read and write
+objects in Pravega similar to reading and writing objects from
 files.
 
-Every Event has a Routing Key.  A Routing Key allows Pravega
-and application developers to reason about which Events are related.   A Routing
-Key is just a string that developers use to group similar Events together. A
-Routing Key is often derived from data naturally occurring in the Event,
-something like "customer-id" or "machine-id", but it could also be some
-artificial String. A
-Routing Key could be something like a date (to group Events together by
-time) or perhaps a Routing Key could be a IoT sensor id (to group Events by
-machine).  A Routing Key is important to defining the precise read and write
-semantics that Pravega guarantees, we will get into that detail a bit later,
-after we have introduced a few more key concepts.
+Every event has a **Routing Key**. A routing key is a string used by developers to group similar events. A routing key is often derived from data naturally occurring in the event,
```

Comment:
I have doubts about how the concept of routing key is described here (in the old version too). Because in Pravega, conversely to messaging systems, routing keys are used to linearize writes to achieve strict ordering. So, I would check with others whether it is necessary or not to change the technical explanation here.

Reply:
Accepted Suggestion

Reply:
retained the new content!